tinygrad/renderer/cstyle.py

```diff
@@ -172,9 +172,11 @@ class ClangRenderer(CStyleLanguage):
 
   # language options
   buffer_suffix = " restrict"
-  type_map = {dtypes.bool:"_Bool", dtypes.half:"__fp16"}
+  type_map = {dtypes.bool:"_Bool", dtypes.half:"__fp16", dtypes.char:"signed char"}
```

Comment:
I am not sure weather i should do it here or in dtype.py because despite being implementation defined almost everything has char signed by default

Reply:
This doesn't make sense